src/emmy/mathbox/plot.clj

```diff
@@ -727,3 +727,10 @@
   (let [[f-bind opts] (mc/compile-3d opts :f 3)]
     (-> (c/wrap [f-bind] ['emmy.mathbox.plot/VectorField opts])
         (ev/fragment scene))))
+(defn implicit-surface
+  [opts]
+  (let [[f-bind opts] (mc/compile-3d opts :f 3)]
```

Comment:
@sritchie -- I'm getting caught up in this function. I know that `mc/compile-3d` is not what I want. But I'm murky on what's going on here -- these look like functions that take the clojure functions, turn them into clojurescript functions and return a fragment that has all the necessary details for a mathbox plot. 


I _think_ I need to implement a `compile-implicit` function that will take the Lhs and Rhs of the implicit function and do a bunch of the same stuff as `compile-3d`. Could you walk me through compile-3d and probably the `compile-state-fn` therein? I don't have a great high-level idea.

Reply:
so what is going on here is that we have a clojure function on the clj side that we need to turn into something like `(js/Function. "input" "body")` for the cljs side.

I can definitely help with this; I think the easiest thing to do now is to just IGNORE this part and then pass in a quoted function, like `'(fn [in out] ....)` with whatever signature you need for your code to work.

Then once everything is humming we can add this part in, which will remove the need for quotes and make everything faster.

wdyt?

Reply:
Willing to try! I don't really know the tradeoffs, it seems like the development path will be speedier, and I'll probably learn something in the process.